frontend/app/routes/$lang+/_protected+/stub-sin-editor.tsx

```diff
@@ -78,7 +78,7 @@ export async function action({ context: { session }, params, request }: ActionFu
     birthdate: '2000-01-01',
     iss: 'GC-ECAS-MOCK',
     locale: 'en-CA',
-    sid: '00000000-0000-0000-0000-000000000000',
+    sid: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
```

Comment:
This needs to be dynamic. Can you add a TODO so that we don't forget to implement this in a future PR?